zenburn.css

```diff
@@ -150,19 +151,19 @@ body {
 #windows #form .input {
   background-color: #434443 !important;
   border-color: #101010 !important;
-  color: #dcdccc !important;
+  color: var(--body-color) !important;
```

Comment:
I'm not sure but you might want to experiment: I wouldn't be surprised if all those `!important` are not required anymore (might need more specificity in the selector).

Reply:
Yeah, I gotta check those out. It's a bit annoying to have so many of them everywhere. I'd also like to go through all the colors and add the missing ones to the comment block at the top. Contributing some more CSS variables to the default theme would be nice too :)